EIPS/eip-6268.md

```diff
@@ -0,0 +1,94 @@
+
+For the token identifier `_id` that is marked as `locked`, `locked(_id)` MUST return the constant value true and any functions that try transferring the token, including `safeTransferFrom` and `safeBatchTransferFrom` function MUST throw.
+
+```solidity
+// SPDX-License-Identifier: CC0-1.0
+pragma solidity ^0.8.0;
+
+interface UNTI {
+  /// @notice Either `LockedSingle` or `LockedBatch` MUST emit when the locking status is changed to locked.
+  /// @dev If a token is minted and the status is locked, this event should be emitted.
+  /// @param _id The identifier for a token.
+  event LockedSingle(uint256 _id);
+
+  /// @notice Either `LockedSingle` or `LockedBatch` MUST emit when the locking status is changed to locked.
+  /// @dev If a token is minted and the status is locked, this event should be emitted.
+  /// @param _ids The list of identifiers for tokens.
+  event LockedBatch(uint256[] _ids);
+
+  /// @notice Either `UnlockedSingle` or `UnlockedBatch` MUST emit when the locking status is changed to unlocked.
+  /// @dev If a token is minted and the status is unlocked, this event should be emitted.
+  /// @param _id The identifier for a token.
+  event UnlockedSingle(uint256 _id);
+
+  /// @notice Either `UnlockedSingle` or `UnlockedBatch` MUST emit when the locking status is changed to unlocked.
+  /// @dev If a token is minted and the status is unlocked, this event should be emitted.
+  /// @param _ids The list of identifiers for tokens.
+  event UnlockedBatch(uint256[] _ids);
+
+
+  /// @notice Returns the locking status of the token.
+  /// @dev SBTs assigned to zero address are considered invalid, and queries
+  /// about them do throw.
+  /// @param _id The identifier for a token.
+  function locked(uint256 _id) external view returns (bool);
+
+  /// @notice Returns the locking statuses of the multiple tokens.
+  /// @dev SBTs assigned to zero address are considered invalid, and queries
+  /// about them do throw.
+  /// @param _ids The list of identifiers for tokens
+  function lockedBatch(uint256[] _ids) external view returns (bool);
+}
+```
+
+## Rationale
+
+There are some proposed uses of UNTransferability Indicator for [EIP-1155](./eip-1155.md). One example is the "Resident Bound Tokens", which indicates that he or she lives in the country or prefecture, and helps creating the local community or providing the government services. Another example is the lecture credit which doesn't need to distinguish the attendees.
+
+This standard will introduce the fungible but non-transferable token, and also introduce the account-bound attributes, and then redefine the definition of SBTs.
+
+## Backwards Compatibility
+
+This proposal is fully backward compatible with [EIP-1155](./eip-1155.md).
+
+## Security Considerations
+
+There are no security considerations related directly to the implementation of this standard.
```

Comment:
```suggestion
Needs discussion.
```
I agree that there are probably not any security considerations, but while EIPs are in draft, this lack of definite knowledge should be indicated.